WeightProducer/python/MCSampleValues.py

```diff
@@ -999,141 +817,164 @@ class MCSampleValuesHelper():
         },
         "ST_s-channel_4f_leptonDecays" : {
             "CrossSection" : XSValues(
-                XS_2016=3.365, XSSource_2016="XSDB (unknown)",
-                XS_2017=3.74,  XSSource_2017="XSDB (unknown)",
-                XS_2018=3.740, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=3.549, XSSource_13TeV="XSDB (unknown)",
             ),
         },
         "ST_s-channel_4f_InclusiveDecays" : {
             "CrossSection" : XSValues(
                 XS_13TeV=10.12, XSSource_13TeV="XSDB (unknown)",
             ),
         },
+        #TODO
         "ST_t-channel_top_4f_InclusiveDecays" : {
             "CrossSection" : XSValues(
                 XS_2016=136.02, XSSource_2016="Unknown (GenXSecAnalyzer/XSDB return 0.0)",
                 XS_2017=113.3,  XSSource_2017="XSDB (NLO)",
                 XS_2018=115.3,  XSSource_2018="GenXSecAnalyzer",
             ),
         },
+        #TODO
         "ST_t-channel_antitop_4f_InclusiveDecays" : {
             "CrossSection" : XSValues(
                 XS_2016=80.95, XSSource_2016="Unknown (GenXSecAnalyzer/XSDB return 0.0)",
                 XS_2017=67.91, XSSource_2017="XSDB (NLO)",
                 XS_2018=69.09, XSSource_2018="GenXSecAnalyzer",
             ),
         },
+        "ST_t-channel_antitop_5f_InclusiveDecays" : {
+            "CrossSection" : XSValues(
+                XS_13TeV=71.74, XSSource_13TeV="XSDB (NLO)",
+            ),
+        },
+        "ST_t-channel_top_5f_InclusiveDecays" : {
+            "CrossSection" : XSValues(
+                XS_13TeV=119.7, XSSource_13TeV="XSDB (NLO)",
+            ),
+        },
         "ST_tW_top_5f_NoFullyHadronicDecays" : {
             "CrossSection" : XSValues(
-                XS_2016=38.09, XSSource_2016="XSDB (NLO)",
-                XS_2017=34.91, XSSource_2017="XSDB (NLO)",
-                XS_2018=34.91, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=32.45, XSSource_13TeV="XSDB (NLO)",
             ),
             "BranchingRatio" : BRValues(
                 BR_13TeV=(1-(1-3*0.105)**2), BRSource_13TeV="http://pdg.lbl.gov/2019/reviews/rpp2018-rev-top-quark.pdf",
             ),
         },
         "ST_tW_antitop_5f_NoFullyHadronicDecays" : {
             "CrossSection" : XSValues(
-                XS_2016=38.06, XSSource_2016="XSDB (NLO)",
-                XS_2017=34.97, XSSource_2017="XSDB (NLO)",
-                XS_2018=34.97, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=32.51, XSSource_13TeV="XSDB (NLO)",
```

Comment:
Do you know why the UL `ST_tW` cross sections are systematically lower than any of the EOY cross sections?